src/browser/web-ai/chatgpt-response-dom.ts

```
@@ -12,60 +15,396 @@ export const CHATGPT_ASSISTANT_SELECTORS: string[] = [
+            ? node
+            : node.querySelector?.('[data-testid^="conversation-turn"]');
+        return {
+            text: textOf(node),
+            messageId: messageNode?.getAttribute?.('data-message-id') || null,
+            turnId: turnNode?.getAttribute?.('data-testid') || null,
+        };
+    };
+
+    let wrappedNodes: AnyNode[] = [];
+    try {
+        const resolver = resolverSource ? (0, eval)(`(${resolverSource})`) : null;
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)